src/test/resources/one.properties

```diff
@@ -1,23 +1,17 @@
-plugin.zipMapper.class = org.icatproject.ids.storage_test.ZipMapper
-
-plugin.main.class = org.icatproject.ids.storage_test.MainFileStorage
-plugin.main.dir = ${HOME}/data/ids/main/
-
-cache.dir = ${HOME}/data/ids/cache/
-
-preparedCount = 100
-processQueueIntervalSeconds = 2
+plugin.zipMapper.class=org.icatproject.ids.storage_test.ZipMapper
+plugin.main.class=org.icatproject.ids.storage_test.MainFileStorage
+plugin.main.dir=${HOME}/data/ids/main/
+cache.dir=${HOME}/data/ids/cache/
+preparedCount=100
+processQueueIntervalSeconds=2
```

Comment:
Same as for DataSelectionDevTest.properties, the original with additional white space is easier to read.